crates/astria-sequencer-relayer/src/data_availability.rs

```diff
@@ -160,7 +164,8 @@ impl NamespaceData for SequencerNamespaceData {}
 #[derive(Serialize, Deserialize, Debug)]
 pub struct RollupNamespaceData {
     pub(crate) block_hash: Hash,
-    pub(crate) chain_id: Vec<u8>,
+    #[serde(with = "sequencer_types::serde::chain_id")]
```

Comment:
why put this here instead of implementing on the `ChainId` type?

Reply:
I did not want to serde as yet another dependency to astria-proto for something that (at the moment) is sequencer-relayer specific. It will probably end up there, but confining this change to just sequencer-relayer seemed like the right choice.

Reply:
Resolved by removing `ChainId` for now.